src/ui/components/Library/Team/View/Tests/Overview/TestOverviewContent.tsx

```diff
@@ -13,19 +15,9 @@ export function TestOverviewContent() {
 
   let children = null;
 
-  if (test) {
-    if (!hasFilters || tests.find(test => test.testId === testId)) {
-      children = (
-        <div className="flex flex-col" data-test-id="TestRunSummary">
-          <div className="flex flex-row items-center justify-between gap-1 border-b border-themeBorder py-2 px-4">
-            <div className="overflow-hidden overflow-ellipsis whitespace-nowrap text-lg font-medium">
-              {test.title}
-            </div>
-          </div>
-          <Stats failureRates={test.failureRates} />
-          <TestErrorList errorFrequency={test.errorFrequency} executions={test.executions} />
-        </div>
-      );
+  if (testId && test) {
+    if (!hasFilters) {
+      children = <TestOverview testId={testId} />;
```

Comment:
This looks broken to me.

I think this means if you type text to filter the list, then select a test, the 3rd column will show nothing.

Why `!hasFilters`

Reply:
This was unintentional — it's fixed now!